### PR TITLE
Adding grims halloween set

### PR DESCRIPTION
**Pull request by XxSARGExX**
This PR contains all the Equipment and Effects needed for the Halloween Seasonal Events

Additions:
- GRIM`S RIPPER
- GRIM`S DISSOLVER
- GRIM´S ENGINE (L)
- GRIM`S ENGINE (H)

Effects and Sounds included

### CODE REVIEW

**Comment by Caenen** [comment]
I see no changes to lootprops for these new equipment items, may lead to crashes. Please check that out.

**Comment by XxSARGExX** [comment]
GRIM´S SET will not be droped by NPC´s

File: DATA/EQUIPMENT/weapon_equip.ini [hunk 1]
```diff
@@ -50434,3 +50434,135 @@ auto_turret = false
...
+; Type: $classLong $damageType
+; Hull/Shield Damage: $hullDamage/$energyDamage
+; Range: $range m
+; Projectile Speed: $speed m/s
+; Refire Rate: $refireRate /s
+; Power Draw: $powerUsage
+; Mass: $mass
+; Integrity: $hitpoints
+; \
+; Trick or Treat!.
```
Comment (erikroe) [inline DATA/EQUIPMENT/weapon_equip.ini:50475]: With ! or with . ?

File: DATA/MISSIONS/lootprops.ini [hunk 2]
```diff
@@ -2582,3 +2582,19 @@ drop_properties = 2, 0, 1, 0, 2, 1
 nickname = fc_rh_mineral_mining_device01
 drop_properties = 2, 0, 1, 0, 2, 1
 
+[mLootProps]
+nickname = halloween_br_gun_tachyon_light01
+drop_properties = 1, 0, 1, 0, 2, 1
+
+[mLootProps]
+nickname = halloween_br_gun_tachyon_medium01_ammo
```
Comment (erikroe) [inline DATA/MISSIONS/lootprops.ini:2590]: Ammo?

File: DATA/MISSIONS/lootprops.ini [hunk 3]
```diff
@@ -2582,3 +2582,19 @@ drop_properties = 2, 0, 1, 0, 2, 1
 nickname = fc_rh_mineral_mining_device01
 drop_properties = 2, 0, 1, 0, 2, 1
 
+[mLootProps]
+nickname = halloween_br_gun_tachyon_light01
+drop_properties = 1, 0, 1, 0, 2, 1
```
Comment (erikroe) [inline DATA/MISSIONS/lootprops.ini:2587]: Is it intentional the drop chance should never exceed 1 item? Usually it is 2.

**Review by erikroe — CHANGES_REQUESTED** [review]

**Review by erikroe — APPROVED** [review]

**Pull request merged**